src/css/parse.js

```diff
@@ -3257,8 +3257,12 @@ function getNamespacePrefix() {
 function checkNamespaceSeparator(i) {
   if (i >= tokensLength) return 0;
 
-  if (tokens[i].type === TokenType.VerticalLine) return 1;
-  else return 0;
+  if (tokens[i].type === TokenType.VerticalLine) {
+    // Return false if `|=` - [attr|=value]
+    return (tokens[i + 1] && tokens[i + 1].type === TokenType.EqualsSign) ? 0 : 1;
+  }
```

Comment:
I find using only of style of either `if … else …` or ternary `… ? … : …` much easier to read:
```js
if (tokens[i].type === TokenType.VerticalLine &&
    tokens[i + 1] && tokens[i + 1].type !== TokenType.EqualsSign) {
  return 1;
} else {
  return 0;
}
```
I assume here that there must be a token after `|` for it to be parsed as namespace separator, otherwise this should be a parsing error.